.vscode/settings.json

```
@@ -1,8 +1,23 @@
 {
+    // formatting style this project adheres to: https://en.wikipedia.org/wiki/Indentation_style#Allman
+    "powershell.codeFormatting.preset": "Allman",
+    "powershell.codeFormatting.openBraceOnSameLine": false,
     "powershell.codeFormatting.useConstantStrings": true,
     "powershell.codeFormatting.useCorrectCasing": true,
+    "powershell.codeFormatting.newLineAfterOpenBrace": true,
+    "powershell.codeFormatting.addWhitespaceAroundPipe": true,

```

```suggestion
    "powershell.codeFormatting.addWhitespaceAroundPipe": true,
    "powershell.codeFormatting.pipelineIndentationStyle": "IncreaseIndentationForFirstPipeline",
```
To match the PSScriptAnalyzer rules, `pipelineIndentationStyle` should be set.